### PR TITLE
Dockerize Applet for testing convenience

### PR DESCRIPTION
**Pull request by jdk24**

### CODE REVIEW

File: raster_messenger/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM maven:3.6.0-jdk-8-slim
+LABEL maintainer="Julian Kaeflein<julian.kaeflein@geomer.de>"
+
+ENV UM_SERVER="host.docker.internal:9876"
+ENV UM_CHANNEL="raster_data"
+ENV TEST_REGION="TestRegion"
+ENV TEST_POLLUTANT="TestSchad"
+
+RUN apt-get update && apt-get -y install -qq cron
```
Comment (chrismayer) [inline raster_messenger/Dockerfile]: Necessary to install `cron`?

**Review by chrismayer — APPROVED** [review]
LGTM, just one minor remark

**Review by JakobMiksch — APPROVED** [review]
Looks alright. One minor comment inline.

File: raster_messenger/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+COPY src/ src/
+COPY pom.xml pom.xml
+
+# install local dependency for UM Java client
+RUN mvn install:install-file -Dfile=local-deps/nClient.jar -DgroupId=com.pcbsys.nirvana.client -DartifactId=sample -Dversion=1.0 -Dpackaging=jar
+
+# build the executable JAR file
+RUN mvn clean package -DskipTests
+
+CMD umchannel=$UM_CHANNEL umserver=$UM_SERVER testRegion=$TEST_REGION testPollutant=$TEST_POLLUTANT java -jar target/test-messenger-jar-with-dependencies.jar
```
Comment (JakobMiksch) [inline raster_messenger/Dockerfile]: I find it confusing that the variable names are not uppercase.

File: raster_messenger/Dockerfile (hunk 1 above)
Comment (jdk24) [reply raster_messenger/Dockerfile]: No, just the usual: 

&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;Good
&ensp;&ensp;&ensp;/&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;&ensp;\\
 Fast &ensp; &ensp; \__me\__ &ensp;&ensp;  Cheap

Will commit after all comments resolved.

File: raster_messenger/Dockerfile (hunk 2 above)
Comment (jdk24) [reply raster_messenger/Dockerfile]: As in 
```suggestion
CMD UMCHANNEL=$UM_CHANNEL ...
```
 for Java args?

Comment (JakobMiksch) [reply raster_messenger/Dockerfile]: you probably would need to change that in the Java program as well.

I just remember that environment variables are always uppercase. 

But we can just keep it if changing has to many side effects. This was just an observation and is not blocking  ...

**Comment by jdk24** [comment]
Changes applied. 
Mind the run params now in uppercase, additional param OFFSETHRS.
Use OFFSETHRS=1 to send message for one hour in the future, so to be able to send more than current timestamp for testing

**Pull request merged**